frontend/src/components/TracingIntegration/TracesComponent.tsx

```diff
@@ -199,14 +199,14 @@ class TracesComp extends React.Component<TracesProps, TracesState> {
   };
 
   private onTracesUpdated = (traces: JaegerTrace[], jaegerServiceName: string) => {
-    const newState: Partial<TracesState> = { traces: traces, jaegerErrors: undefined, toolbarDisabled: false };
+    const newState: Partial<TracesState> = { traces: traces, tracingErrors: undefined, toolbarDisabled: false };
     if (this.state.targetApp === undefined && jaegerServiceName) {
       newState.targetApp = jaegerServiceName;
     }
     this.setState(newState as TracesState);
   };
 
-  private getJaegerUrl = () => {
+  private getTracingUrl = () => {
     if (this.props.urlJaeger === '' || !this.state.targetApp) {
```

Comment:
if the method name is getTracingUrl, the property should be renamed to urlTracing (`this.props.urlTracing`)